census/src/main/scala/grasshopper/census/search/AddressInterpolator.scala

```diff
@@ -38,33 +24,15 @@ object AddressInterpolator {
     AddressRange(start, end)
   }
 
-  private def rangeIsEven(f: Feature, isLeft: Boolean): Boolean = {
-    val prefix = if (isLeft) "L" else "R"
-    val fromEven = f.values.getOrElse(s"${prefix}FROMHN", 0)
-    val fromRange = fromEven match {
-      case "" => 0
-      case _ => fromEven.toString.toInt
-    }
-    val fromIsEven = fromEven != "" && fromRange % 2 == 0
-
-    val toEven = f.values.getOrElse(s"${prefix}TOHN", 0)
-    val toRange = fromEven match {
-      case "" => 0
-      case _ => toEven.toString.toInt
-    }
-    val toIsEven = toEven != "" && toRange % 2 == 0
-    fromIsEven || toIsEven
-  }
-
   def interpolate(feature: Feature, range: AddressRange, a: Int): Feature = {
-    val sign = if (a % 2 == 0) -1 else 1
+    val pre = prefix(feature, a)
+    val sign = if (pre == "R") -1 else 1
     val line = feature.geometry.asInstanceOf[Line]
     val l = line.length
     val d = calculateDistance(range)
     val x = a - range.start
     val dist = x * l / d
-    //TODO: Review how offset is being calculated
-    val geometry = line.pointAtDistWithOffset((dist * -1), sign * 0.0001)
+    val geometry = line.pointAtDistWithOffset(dist, sign * 0.0001)
```

Comment:
What is the `0.0001` value?  I believe its the offset distance from the interpolated point, but it be more clear with a well-named variable or a comment.